cmd/pro/import_workspace.go

```diff
@@ -180,9 +183,14 @@ func resolveInstanceOptions(ctx context.Context, instance *managementv1.DevPodWo
 	if err != nil {
 		return nil, fmt.Errorf("find template: %w", err)
 	}
-	templateParameters, err := list.GetTemplateParameters(template, instance.Spec.TemplateRef.Version)
-	if err != nil {
-		return nil, fmt.Errorf("get template parameters: %w", err)
+	var templateParameters []storagev1.AppParameter
+	if len(template.Spec.Versions) > 0 {
+		templateParameters, err = list.GetTemplateParameters(template, instance.Spec.TemplateRef.Version)
+		if err != nil {
+			return nil, fmt.Errorf("get template parameters: %w", err)
+		}
+	} else {
+		templateParameters = template.Spec.Parameters
```

Comment:
feel free to disregard: what do you think about using this as default and overwriting with `list.GetTemplateParameters` result if `len(template.Spec.Versions) > 0`?  Or you prefer to keep declaration and initialisation of this var separate?

Reply:
that's way better, not sure what I was thinking. Ty!